From 1051b34db65a45a4fd5f12cad0eebc27325a2be3 Mon Sep 17 00:00:00 2001
From: Jonathan Riddell <jr@jriddell.org>
Date: Fri, 19 Jun 2020 15:35:08 +0100
Subject: [PATCH] switch order of cmake modules so it gives priority to the
 local copy which will be the one it expects.  fixes build failure on KDE neon

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 00ee0035..5b96f25b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,8 +20,8 @@ endif(POLICY CMP0063)
 set(CMAKE_INCLUDE_CURRENT_DIR TRUE)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH}
-                      ${ECM_MODULE_PATH}
-                      ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules)
+                      ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules
+                      ${ECM_MODULE_PATH})
 
 #include(ECMAddTests)
 #include(ECMMarkNonGuiExecutable)
-- 
GitLab

